app/assets/stylesheets/homepage.css.scss

```diff
@@ -16,14 +16,39 @@
     font-weight: 200;
 }
 
-.nav-bar-nk {
+#logo {
+    text-decoration: none;
+    margin-left: -70px ;
+}
+
+.nav-bar-link {
     color: white;
-    font-size: 20px;
+    font-size: 1.1em;
+    text-decoration: none;
+    height: 76px;
+    display: block;
+    padding: 8px 12px 0 12px;
+}
+
+.nav-bar-link:hover {
+  background-color: rgba(0,0,0,.4);
```

Comment:
`.4` should be written with a leading zero as `0.4`<br>Commas in function arguments should be followed by a single space